### PR TITLE
Removing Login/Logout/Register UI, Code Cleanup

### PR DESCRIPTION
**Pull request by kg185155**
## Description
This PR hides the Login/Logout/Register UI elements from across the site.
It also does some code cleanup and organization:
1. Files have been formatted with `prettier`
2. The `..menu.html` files are now extending `base.html` instead of including the JS and menu every time. This will reduce chances of missing changes made to the header/navigation.
3. Fixing of error on homepage of button on submit function not found

## Screenshots/Screencasts
See: https://recordit.co/opi4Mo5Jpf (hopefully that works)

## Testing
1. Check out this branch and start the site
2. Search for stores
3. Visit each location and confirm filtering, add to cart, etc. still works
4. Visit the cart
5. Confirm adding items, removing items from cart still works
6. Checkout
7. Confirm taken to confirmation screen
8. Visit about page
9. Confirm documentation in about still works

## Ticket
#15

### CODE REVIEW

File: code/BurgersUnlimited/urls.py [hunk 1]
```diff
@@ -23,15 +23,14 @@
 from django.urls import include, path
 from django.views.generic import RedirectView
 
-from users import views as user_v
-
+# from users import views as user_v
```
Comment (cb185224) [inline code/BurgersUnlimited/urls.py:26]: Please don't commit commented code, just delete it; it'll be in the git history if we really need it.

File: code/BurgersUnlimited/urls.py [hunk 2]
```diff
@@ -23,15 +23,14 @@
...
     path('burger/', include('burger.urls'), name='main-burger'),
     path('', RedirectView.as_view(url='burger/')),
     path('__debug__/', include(debug_toolbar.urls)),
-    path('Register/', user_v.register, name='Register'),
-    path('login/', auth_v.LoginView.as_view(template_name='users/login.html'), name='login'),
-    path('logout/', auth_v.LogoutView.as_view(template_name='users/logout.html'), name='logout'),
-    ] + static(settings.STATIC_URL, document_root = settings.STATIC_ROOT)
+    # path('Register/', user_v.register, name='Register'),
+    # path('login/', auth_v.LoginView.as_view(template_name='users/login.html'), name='login'),
+    # path('logout/', auth_v.LogoutView.as_view(template_name='users/logout.html'), name='logout'),
```
Comment (cb185224) [inline code/BurgersUnlimited/urls.py:35]: Please don't commit commented code, just delete it; it'll be in the git history if we really need it.

**Review by cb185224 — CHANGES_REQUESTED** [review]

File: code/BurgersUnlimited/urls.py (hunk 1 above)
Comment (kg185155) [reply code/BurgersUnlimited/urls.py]: Updated! Thank you for the review.

File: code/BurgersUnlimited/urls.py (hunk 2 above)
Comment (kg185155) [reply code/BurgersUnlimited/urls.py]: Updated! Thank you for the review.

**Review by cb185224 — APPROVED** [review]

**Pull request merged**